src/api/custom-resolvers/mutations/update-exporter-company-and-company-address.ts

```diff
@@ -17,7 +17,7 @@ const updateExporterCompanyAndCompanyAddress = async (
 ): Promise<ApplicationRelationship> => {
   try {
     console.info('Updating application exporter company and exporter company address for ', variables.companyId);
-    const { address, sicCodes, oldSicCodes, ...exporterCompany } = variables.data;
+    const { address, sicCodes, sicCodeDescriptions, oldSicCodes, ...exporterCompany } = variables.data;
```

Comment:
Not sure how this is generated but could we be consistant with the `industryName` naming?